docs/How-to/Manage-a-Terra-validator/Set-up-validator.md

```diff
@@ -2,18 +2,19 @@
 
 This is a detailed step-by-step guide for setting up a Terra validator. Please be aware that while it is easy to set up a rudimentary validating node, running a production-quality validator node with a robust architecture and security features requires an extensive setup.
 
-::: tip
-Block42 has put together an excellent [step-by-step guide](https://medium.com/block42-blockchain-company/how-to-setup-a-terra-luna-validator-node-860d8ea7aea2) for setting up a new validator.
+::: tip Additional resources
+To supplement this guide, consider using Block42's helpful [step-by-step guide](https://medium.com/block42-blockchain-company/how-to-setup-a-terra-luna-validator-node-860d8ea7aea2) for setting up a new validator.
 :::
 
-## Requirements
+## Prerequisites
 
-This guide starts with the following assumptions:
+- You have completed [how to run a full Terra node](https://docs.terra.money/How-to/Run-a-full-Terra-node/Hardware-requirements.html).
```

Comment:
add "Which includes the following steps:"

As it is it sounds like they have to follow the terra full node guide, and the additional bullet points.